tests/server.rs

```diff
@@ -57,7 +57,7 @@ async fn validate_component_package(config: &Config, client: &FileSystemClient)
         .context("failed to resolve package")?;
     assert_eq!(
         download.digest.to_string(),
-        "sha256:396bf81fe30c615180c31fc3ba964396241af472ace265f55609a3fcf12140f2"
+        "sha256:24eaf3439a60c96764c4f2c92a0a81b52bd4fe5d5b91f090184a0f6486b1bf29"
```

Comment:
Thoughts on deriving this hash automatically from the content so that it doesn't have to be changed when the content does?

Reply:
I can add that.